### ISSUE TITLE
Add examples

### ISSUE THREAD

**Issue by prisae** [empymod/empymod#64]
Add more examples:

- Figures 4.7 and 4.8 to https://empymod.readthedocs.io/en/stable/examples/reproducing/fem_tem.html#sphx-glr-examples-reproducing-fem-tem-py
- Add examples for the following layouts:
  - central loop
  - loop loop
  - concentric loop
- A examples for ramp and arbitrary waveform; pulses; filtering.

**Comment by leonfoks** [comment]
I think system specific examples would help too, these would be "higher level" than your proposed examples above. Each example could identify where a user might have to change parameters to match their specific data set, like waveform, area, offset etc.
* SkyTEM
* Aeroquest
* WalkTEM
* Resolve

**Comment by prisae** [comment]
Yes, that would be great! However, I personally rely here on the expertise of others to implement -or help me with- that, as I have no idea about any of these systems.
